src/Fantomas.Tests/TriviaTests.fs

```diff
@@ -541,3 +542,33 @@ let x =
     | [ { Type = TriviaNodeType.MainNode SynConst_Unit
           ContentBefore = [ Directive "#if DEBUG\n\n#endif" ] } ] -> pass ()
     | _ -> Assert.Fail(sprintf "Unexpected trivia: %A" trivia)
+
+[<Test>]
```

Comment:
This isn't really the correct place for this unit test, please move it to: `CommentTests.fs`

Reply:
It should be better now. Please double check! :)